Source/Transforms/LMCHilbertTransform.cc

```diff
@@ -65,6 +100,14 @@ namespace locust
     	return fbufferMargin;
     }
 
+    bool HilbertTransform::SetupHilbertTransform()
+    {
+        originaldata = (fftw_complex *) fftw_malloc(sizeof(fftw_complex) * fbufferSize);
+        SignalComplex = (fftw_complex*)fftw_malloc( sizeof(fftw_complex) * fbufferSize );
+        FFTComplex = (fftw_complex*)fftw_malloc( sizeof(fftw_complex) * fbufferSize );
+        hilbert = (fftw_complex *) fftw_malloc(sizeof(fftw_complex) * fbufferSize);
+        return fComplexFFT.SetupFFTWindow(fbufferSize, fWindowName, fWindowParam);
+    }
```

Comment:
This looks great.  Can it be called from within HilbertTransform::Configure() instead of from LMCArraySignalGenerator?

Reply:
Yes, I think so. I'll move it and check.